SPM Example/PortalSwift/Portal.swift

```diff
@@ -167,7 +167,17 @@ class PortalWrapper {
       )
 
       request.send { (_: Result<String>) in
-        completion(Result(data: true))
+        do {
```

Comment:
Was this already in cocoapods?

Reply:
instead of returning true we needed to be calling that method.